macros/models/_sector/dex/uniswap_compatible_trades.sql

```diff
@@ -59,16 +59,18 @@ FROM
     , version = null
     , Pair_evt_Swap = null
     , Factory_evt_PoolCreated = null
+    , taker_column_name = 'recipient'
     , optional_columns = ['f.fee']
+    , pair_column_name = 'pool'
     )
 %}
 WITH dexs AS
 (
     SELECT
         t.evt_block_number AS block_number
         , t.evt_block_time AS block_time
-        , t.recipient AS taker
-        , CAST(NULL as VARBINARY) as maker
+        , t.{{ taker_column_name }} AS taker
+        , t.contract_address as maker
```

Comment:
Hi @jeff-dude,
can we put `t.contract_address` as maker instead of null in the `uniswap_compatible_v3_trades` macro? 
Like maker in the `uniswap_compatible_v2_trades`.

Reply:
have you seen [this](https://github.com/duneanalytics/spellbook/pull/5129/files#diff-b4b0984ff6cb46908aa47074c4a8d539e4c5f0c5b58e6997a14a478b85ba795dR66-R70) logic for `maker` from @tomfutago?

would that work here? (sorry, haven't dug in too deep outside of your specific question)

Reply:
That approach would work here, thanks.

Reply:
I am just wondering should we make `uniswap_v2` and `uniswap_v3` consistent in `maker` field.

Reply:
i don't think we need to force it, until the data tells us to update v2